### ISSUE TITLE
How to get the Memory usage by process matching with Task manager

### ISSUE THREAD

**Issue by YangLu1031** [martinlindhe/wmi_exporter#496]
Tried `wmi_process_working_set`, `wmi_process_virtual_bytes` and `wmi_process_private_bytes` metrics under the [process collector](https://github.com/martinlindhe/wmi_exporter/blob/master/docs/collector.process.md), but none of them match with the result of Task manager.

i.e. powershell mem usage from task manager is 32.1MB, but 
`wmi_process_working_set` -> 73285632
`wmi_process_virtual_bytes` -> 2204049985536
`wmi_process_private_bytes` -> 87359488

**Comment by YangLu1031** [comment]
cc @pjh

**Comment by carlpett** [comment]
Hi @YangLu1031,
As @Mario-Hofstaetter explained, there are many different memory usage metrics that can be displayed. The default one is called "active private working set", which is not exposed directly via WMI or performance counters, so it is likely something which taskmgr calculates internally.
I would suggest doing what @Mario-Hofstaetter is doing and paying attention to private_bytes and working_set, and possibly adjusting your taskmgr to display the same columns if you need to cross-reference.
Also pay attention to that when Task Manager displays eg 48,856K, that is with 1024 bytes to a K. The metrics as you see them in the raw metrics output is in bytes, eg 5.0085888e+07. Those are the same values, although you might need a calculator to verify.

**Comment by YangLu1031** [comment]
Thanks! @Mario-Hofstaetter  @carlpett

**Comment by makedir** [comment]
I found by Google search to this post, because no other post I was able to find anywhere, would give a clear answer to this. I have the following issue which is kinda bit of a puzzle to me. That Task Manager RAM usage is totally different, than what Process Explorer shows.

A good example is the process dwm.exe with these examples from the same time stamp:

Process Explorer:
![image](https://user-images.githubusercontent.com/6721083/142299086-4135c52f-c16d-496e-b221-32b6b2e3ca3d.png)

Task Manager
![image](https://user-images.githubusercontent.com/6721083/142299111-34a142cd-e4c7-46ee-96eb-f27f1dfa8855.png)

Those values dont make any sense, regardless how you calculate them.

The problem also is, that in Process Explorer the "private bytes" for dwm.exe grows over time, to sizes like x GB in size, even though Task Manager still shows something like 60-120MB.
